packages/optimism/src/Optimism.ts

```diff
@@ -0,0 +1,54 @@
+import { type BridgeActionParams, compressJson } from '@rabbitholegg/questdk'
+import {
+  l1StandardBridgeABI,
+  l2StandardBridgeABI,
+  addresses,
+} from '@eth-optimism/contracts-ts'
+import { ETH_CHAIN_ID, CHAIN_ID_ARRAY } from './chain-ids.js'
+import { ETH_TOKEN_ADDRESS } from './token-addresses.js'
+// If you're implementing swap or mint, simply duplicate this function and change the name
```

Comment:
can remove the template comments

Reply:
Good catch thanks, I'll keep an eye out for loose comments in the other branches too.